src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+
+#: src/view/screens/AppPasswords.tsx:283
+msgid "Are you sure you want to delete the app password \"{name}\"?"
+msgstr "Weet u zeker dat u het app-wachtwoord \"{name}\" wilt verwijderen?"
+
+#: src/components/dms/MessageMenu.tsx:149
+msgid "Are you sure you want to delete this message? The message will be deleted for you, but not for the other participant."
+msgstr "Weet u zeker dat u dit privébericht wilt verwijderen? Het privébericht wordt voor u verwijderd, maar niet voor de andere deelnemer."
+
+#: src/screens/StarterPack/StarterPackScreen.tsx:632
+msgid "Are you sure you want to delete this starter pack?"
+msgstr "Weet u zeker dat u dit startpakket wilt verwijderen?"
+
+#: src/screens/Profile/Header/EditProfileDialog.tsx:82
+msgid "Are you sure you want to discard your changes?"
+msgstr "Weet u zeker dat u uw wijzigingen ongedaan wilt maken?"
+
+#: src/components/dms/LeaveConvoPrompt.tsx:48
+msgid "Are you sure you want to leave this conversation? Your messages will be deleted for you, but not for the other participant."
+msgstr "Weet u zeker dat u dit gesprek wilt verlaten? Uw privéberichten worden voor u verwijderd, maar niet voor de andere deelnemer."
+
+#: src/components/FeedCard.tsx:313
+msgid "Are you sure you want to remove this from your feeds?"
+msgstr "Weet u zeker dat u dit uit uw feeds wilt verwijderen?"
+
+#: src/view/com/feeds/FeedSourceCard.tsx:313
+msgid "Are you sure you want to remove {0} from your feeds?"
+msgstr "Weet u zeker dat u {0} uit uw feeds wilt verwijderen?"
+
+#: src/view/com/composer/select-language/SuggestedLanguage.tsx:60
+msgid "Are you writing in <0>{0}</0>?"
+msgstr "Schrijft u in <0>{0}</0>?"
+
+#: src/screens/Onboarding/index.tsx:23
+#: src/screens/Onboarding/state.ts:82
+msgid "Art"
+msgstr "Kunst"
+
+#: src/view/com/composer/labels/LabelsBtn.tsx:170
+msgid "Artistic or non-erotic nudity."
+msgstr "Artistieke of niet-erotische naaktheid."
+
+#: src/screens/Signup/StepHandle.tsx:173
+msgid "At least 3 characters"
+msgstr "Minstens 3 tekens"
+
+#: src/components/dms/MessagesListHeader.tsx:75
+#: src/components/moderation/LabelsOnMeDialog.tsx:290
+#: src/components/moderation/LabelsOnMeDialog.tsx:291
+#: src/screens/Login/ChooseAccountForm.tsx:90
+#: src/screens/Login/ChooseAccountForm.tsx:95
+#: src/screens/Login/ForgotPasswordForm.tsx:123
+#: src/screens/Login/ForgotPasswordForm.tsx:129
+#: src/screens/Login/LoginForm.tsx:282
+#: src/screens/Login/LoginForm.tsx:288
+#: src/screens/Login/SetNewPasswordForm.tsx:154
+#: src/screens/Login/SetNewPasswordForm.tsx:160
+#: src/screens/Messages/components/ChatDisabled.tsx:133
+#: src/screens/Messages/components/ChatDisabled.tsx:134
+#: src/screens/Profile/Header/Shell.tsx:80
+#: src/screens/Signup/BackNextButtons.tsx:42
+#: src/screens/StarterPack/Wizard/index.tsx:307
+#: src/view/com/util/ViewHeader.tsx:87
+msgid "Back"
+msgstr "Terug"
+
+#: src/view/screens/Settings/index.tsx:442
+msgid "Basics"
+msgstr "Basis"
```

Comment:
That's not a translation of `Basics`, but `basic`. Here, basics probably means "the basic principles" or "Algemeen". It is shown in the profile settings:

![Profile settings](https://github.com/user-attachments/assets/0a8ed544-1cdb-4dc0-b4b8-8cab7571108f)

Proposed translations for that would be:
- Algemeen
- Basisprincipes
- Grondbeginselen

Reply:
Thanks for context. Might be a bug in the core UI but risk of introducing wrong translation for another context seems low. Applied.